src/smc-util/message.coffee

```diff
@@ -448,7 +481,7 @@ message
 # starting_bundle_number.
 #
 # client --> hub --> project_server
-API message
+API message # unimplemented
```

Comment:
> Why do you say "unimplemented?" It is too implemented.

I got unknown message event errors and didn't see the message event type elsewhere in the source using `git grep`.
```
 {"error":"unknown message event type 'public_get_project_info'"}
 {"error":"unknown message event type 'public_project_info'"}
 {"error":"unknown message event type 'save_project'"}
```